# msg file for task queue

int16 task_id
int16 task_priority
string task_type
string initial_preset_location
string final_preset_location
string part_type
geometry_msgs/Pose part_pose
geometry_msgs/Pose part_target_pose